server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -50,17 +61,25 @@ public class Upgrader12to13 implements Upgrader {
 
   private static final Logger LOG = LoggerFactory.getLogger(Upgrader12to13.class);
 
+  private static final Text COMPACT_QUAL = new Text("compact");
```

Comment:
Using the following will make some other suggestions work.

```suggestion
  private static final ColumnFQ COMPACT_COL = new ColumnFQ(MetadataSchema.TabletsSection.ServerColumnFamily.NAME, new Text("compact"));
```

Reply:
Implemented in 69b3c48